Jenkinsfile

```diff
@@ -28,8 +33,24 @@ fhBuildNode(['label': 'nodejs']) {
 
     stage('Build') {
         gruntBuild {
-            name = 'fh-template-apps'
+            name = COMPONENT
             distCmd = 'default'
         }
+        s3PublishArtifacts([
+                bucket: "fh-wendy-builds/${COMPONENT}/${BUILD}",
+                directory: "./dist"
+        ])
+    }
+
+    stage('Platform Update') {
+        final Map updateParams = [
+                componentName: COMPONENT,
+                componentVersion: VERSION,
+                componentBuild: BUILD,
+                changeUrl: CHANGE_URL
+        ]
+        fhcapComponentUpdate(updateParams)
+        //ToDo This will also need to do something yet TBD for rhmap4
```

Comment:
Opening this PR as is, but we probably want to figure out exactly what we intend to do about rhmap4 updates and add the correct logic here before we merge this.